package.json

```
@@ -740,7 +740,7 @@
     "package": "vsce package",
     "test": "npm run compile:test && node ./out/test/runTest.js",
     "test:unit": "jest",
-    "test:unit:web": "npm run compile && vscode-test-web --headless --browserType=chromium --extensionDevelopmentPath=. --extensionTestsPath=./out/web/test/integration/index.js",
+    "test:unit:web": "npm run compile && npx playwright install chromium && vscode-test-web --headless --browser=chromium --extensionDevelopmentPath=. --extensionTestsPath=./out/web/test/integration/index.js",
```

```suggestion
    "test:unit:web": "npm run compile && vscode-test-web --headless --browser=chromium --extensionDevelopmentPath=. --extensionTestsPath=./out/web/test/integration/index.js",
```

What do you think about adding `@playwright/browser-chromium` as dev dependency instead? This allows for better dependency caching and fixes the `npm run web` command, too.

---

Kinda prefer being explicit, and the install command is idempotent, but can't argue with fire and forget automatic dependency installation